src/store.ts

```diff
@@ -69,6 +71,43 @@ export const DefaultMarkThresholds: Readonly<MarkThresholds> = {
   definite: 0.25,
 }
 
+const leanInterpretation = (
+  interpretation: PageInterpretation | undefined
+): PageInterpretation | undefined => {
```

Comment:
Using a function declaration with overloads allows you to better define the type:
```suggestion
function leanInterpretation(interpretation: undefined): undefined
function leanInterpretation(interpretation: PageInterpretation): PageInterpretation
function leanInterpretation(interpretation: PageInterpretation | undefined): PageInterpretation | undefined {
```